sql/system-database.md

```diff
@@ -34,3 +34,222 @@ Currently, the `help_topic` is NULL.
 
 - `GLOBAL_VARIABLES`: global system variable table
 - `tidb`: to record the version information when TiDB executes `bootstrap`
+
+## INFORMATION\_SCHEMA Tables
+
+To be compatible with MySQL, TiDB supports INFORMATION\_SCHEMA tables. Some third-party software queries information in these tables. Currently, most INFORMATION\_SCHEMA tables in TiDB are null.
+
+### CHARACTER\_SETS Table
+
+The CHARACTER\_SETS table provides information about character sets. But the table data is fake. By default, TiDB only supports utf8mb4.
+
+```sql
+mysql> select * from CHARACTER_SETS;
++--------------------|----------------------|-----------------------|--------+
+| CHARACTER_SET_NAME | DEFAULT_COLLATE_NAME | DESCRIPTION           | MAXLEN |
++--------------------|----------------------|-----------------------|--------+
+| ascii              | ascii_general_ci     | US ASCII              |      1 |
+| binary             | binary               | Binary pseudo charset |      1 |
+| latin1             | latin1_swedish_ci    | cp1252 West European  |      1 |
+| utf8               | utf8_general_ci      | UTF-8 Unicode         |      3 |
+| utf8mb4            | utf8mb4_general_ci   | UTF-8 Unicode         |      4 |
++--------------------|----------------------|-----------------------|--------+
+5 rows in set (0.00 sec)
+```
+
+### COLLATIONS Table
+
+The COLLATIONS table is similar to the CHARACTER\_SETS table.
+
+### COLLATION\_CHARACTER\_SET\_APPLICABILITY Table
+
+Null table.
+
+### COLUMNS Table
+
+The COLUMNS table provides information about columns in tables. The information in this table is not accurate. To query information, it is recommended to use the `SHOW` statement:
+
+```sql
+SHOW COLUMNS FROM table_name [FROM db_name] [LIKE 'wild']
+```
+
+### COLUMNS\_PRIVILEGE Table
+
+Null table.
+
+### ENGINES Table
+
+The ENGINES table provides information about storage engines. Currently, the table data is fake. In the production environment, use the TiKV engine for TiDB.
+
+### EVENTS Table
+
+Null table.
+
+### FILES Table
+
+Null table.
+
+### GLOBAL\_STATUS Table
+
+Null table.
+
+### GLOBAL\_VARIABLES Table
+
+Null table.
+
+### KEY\_COLUMN\_USAGE Table
+
+The KEY\_COLUMN\_USAGE table describes which key columns have constraints.
```

Comment:
-> The KEY\_COLUMN\_USAGE table describes the key constraints of the columns, like whether a column has a primary key.